index.mdx

```diff
@@ -41,6 +41,10 @@ public: true
 
 Sign up for Civic Auth in less than a minute at [auth.civic.com](https://auth.civic.com) to get your Client ID.
 
+<Info>
```

Comment:
I would leave this section out , and add it only in the faq or somewhere less prominent, as it is not necessary to know during "quick start"